packages/EasyErrorHandler/composer.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "eonx-com/easy-error-handler",
+    "description": "Provides customizable ready-to-use error handler for Lumen applications",
+    "type": "library",
+    "license": "MIT",
+    "require": {
+        "php": "^7.1",
+        "eonx-com/externals": "^1.0 || ^2.0",
+        "nesbot/carbon": "^2.22",
+        "eonx-com/easy-logging": "^2.3",
+        "eonx-com/apiformats": "^1.0",
+        "illuminate/support": "5.8.*"
```

Comment:
In dev you require `"laravel/lumen-framework": "^5.5"` so here you should be consistent with the version.

Reply:
done